packages/eslint/src/executors/lint/utility/eslint-utils.ts

```diff
@@ -72,10 +73,26 @@ export async function resolveAndInstantiateESLint(
       options.reportUnusedDisableDirectives || undefined;
   }
 
+  const eslintVersion = getESLintVersion(ESLint.version);
+
+  // pass --quiet to eslint 9+ directly: filter only errors
+  if (options.quiet && eslintVersion.major >= 9) {
```

Comment:
Use `semver` instead of custom parsing logic. It's already a dependency of `@nx/eslint`.

```
import { gte } from 'semver';
//...
if (options.quiet && gte(ESLint.version, '9.0.0') {
  //...
}

Reply:
updated